Display.cpp

```diff
@@ -38,17 +38,34 @@ namespace display {
 void Display::begin() {
   lcd_->begin(kWidth, kHeight);
   lcd_->noCursor(); 
+  addCustomCharacters();
   update();
 }
 
+void Display::addCustomCharacters() {
+  lcd_->createChar(0, patientIcon);
+  lcd_->createChar(1, timeIcon);
+  lcd_->createChar(2, peekIcon);
+  lcd_->createChar(3, plateauIcon);
+  lcd_->createChar(4, peepIcon);
+  lcd_->createChar(5, ieiIcon);
+  lcd_->createChar(6, ie1Icon);
+  lcd_->createChar(7, bellIcon);
+}
+
 void Display::update() {
   writeHeader();
+  writeFooter();
 }
 
-void Display::setAlarmText(const String& alarm) {
-  if (animation_.text() != alarm) {
-    animation_.reset(alarm);
+void Display::setAlarmText(const String& alarmH, const String& alarmF, const int& countdown) {
```

Comment:
This is confusing, snooze countdown is not related to the alarm text.

Reply:
Revised it so that snooze timer text is now passed to the display in a separate display method.